### PR TITLE
Use database

### PR DESCRIPTION
**Pull request by ArchStanton987**
This PR includes :
- update sql script for database creation
- update test_routes.http
- server.js : now uses studips database instead of datas stored in server.js

### CODE REVIEW

File: back/server.js [hunk 1]
```diff
@@ -15,250 +15,61 @@ app.use(
...
+				res.status(400).send('Wrong email or password')
 			}
-			match++
 		}
-	}
-	if (match === 0) {
-		res.send('Wrong username or password')
-	}
-});
+		if (err) {
```
Comment (ghost) [inline back/server.js:59]: Can you ever get to this line of code ?

**Review by ghost — CHANGES_REQUESTED** [review]
Plus some conflicts on server.js to solve

**Comment by ghost** [comment]
Conflicts to resolve (server.js)

File: back/server.js (hunk 1 above)
Comment (ArchStanton987) [reply back/server.js:59]: I assumed we could get to this line of code in case something goes wrong when accessing the database... Should I remove it ?

**Pull request merged**